lms/static/scripts/frontend_apps/components/GradingComment.tsx

```diff
@@ -37,20 +31,21 @@ function CommentPopover({
   const commentRef = useRef<HTMLTextAreaElement | null>(null);
   const commentId = useId();
 
-  // Focus comment textarea when popover is open
-  useLayoutEffect(() => {
-    commentRef.current!.focus();
-  }, []);
-
   return (
-    <div
-      role="dialog"
-      className={classnames(
+    <Dialog
+      variant="custom"
+      title=""
```

Comment:
The `title` prop is required, even though it's not used when `variant` is `"custom"`. Probably something we need to address in frontend-shared, but an empty string does the trick for now.